.app/functions/liquid_filters.bash

```diff
@@ -0,0 +1,148 @@
+# These functions are available within Bash sub-shells, when
+# writing your Markdown. For example, function 'foo' can be
+# used like so:
+#
+#    <?bash echo $var | foo ;?>
+
+# These functions are intended to mimic some of the liquid
```

Comment:
Add note here, that these filters are also available in *most* of the mustache template stuff too (with the exception of the `{{foo.bar}}` stuff inside custom iterators.. which is half the site) and can be used like so:

```
{{some_var | some_filter}}
```